EXT_PRAYTRACE.md

```diff
@@ -0,0 +1,188 @@
+Domain-Specific Naming Conventions for Photon Raytracing Codes
+===========================================================================
+
+openPMD extension name: `PRAYTRACE`
+
+
+Introduction
+------------
+
+This extension to the openPMD addresses the domain-specific requirements
+met in raytracing codes (e.g. OASYS).
+
+Particle Based Records (Fields)
+---------------------------
+
+### Root attributes
+
+- `openPMD`: `0.0.1`
+- `openPMDextension`: `PRAYTRACE`
+- `author`: `Aljosa Hafner <aljosa.hafner@ceric-eric.eu>`
+- `date`: `10.10.2019 00:00:00 +0100`
+
+### Attributes for the Group 'particlesPath'
+
+	- `nRays`
```

Comment:
maybe better suited as an attribute to the top level datagroup? can also be deducted from the data.